packages/core/src/endowmentsToolkit.js

```diff
@@ -263,20 +329,29 @@ function endowmentsToolkit({
     return newPropDesc
   }
 
+  /**
+   *
+   * @param {PropertyDescriptor} sourcePropDesc
+   * @param {object} unwrapFromGlobalThis
+   * @param {object} unwrapToGlobalThis
+   * @returns {PropertyDescriptor}
+   */
   function applyGetSetPropDescTransforms(
     sourcePropDesc,
     unwrapFromGlobalThis,
     unwrapToGlobalThis
   ) {
     const wrappedPropDesc = { ...sourcePropDesc }
     if (sourcePropDesc.get) {
+      const getter = sourcePropDesc.get
       wrappedPropDesc.get = function () {
+        // eslint-disable-next-line @typescript-eslint/no-this-alias
         const receiver = this
         // replace the "receiver" value if it points to fake parent
         const receiverRef =
           receiver === unwrapFromGlobalThis ? unwrapToGlobalThis : receiver
         // sometimes getters replace themselves with static properties, as seen wih the FireFox runtime
-        const result = Reflect.apply(sourcePropDesc.get, receiverRef, [])
+        const result = Reflect.apply(getter, receiverRef, [])
```

Comment:
(must) Let's make sure this is not breaking the code. I'm pretty sure the comment above is explaining why we're not using a local copy but referencing the getter from the original source and this change would break it.

Reply:
Also, this change is already present in the branch which is set as a merge target for this PR. Something broke with the hierarchy there.

Reply:
Whatever the problem here was, it's resolved now.  I'm not sure what happened.